third_party/aws-sdk-cpp/aws-sdk-cpp.bzl

```diff
@@ -0,0 +1,195 @@
+#
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+def aws_sdk_cpp():
+    aws_sdk_cpp_repository(name="_aws_sdk_cpp")
+    new_git_repository(
+        name = "aws-sdk-cpp",
+        remote = "https://github.com/aws/aws-sdk-cpp.git",
+        build_file = "@_aws_sdk_cpp//:BUILD",
+        tag = "1.11.268",
+        init_submodules = True,
+        recursive_init_submodules = True,
+        # https://github.com/bazelbuild/bazel/issues/374
+        # issues with ASCI handling of file_test.c *xample file.txt in bazel
+        patch_cmds = ["find . -name '*xample.txt' -delete"],
```

Comment:
why `*xample` ?

Reply:
Because bazel cannot handle non ascii names:
aws-sdk-cpp/crt/aws-crt-cpp/crt/aws-c-common/tests/resources/Å Éxample.txt
I would have the full name here but I cannot  ;)